templates/index.html

```diff
@@ -17,505 +19,653 @@
+            width="1012",
+            height="297",
             hi_def=True,
-            loading="auto",
-            attrs={"class": "p-logo-section__logo"}
+            loading="auto|lazy"
             ) | safe
           }}
         </div>
-        <div class="p-logo-section__item">
+        <div class="col-9 col-medium-4 col-small-4">
+          <div class="row p-block">
+            <div class="col-6">
+              <h2>More automation and cost savings</h2>
+            </div>
+            <div class="col-3">
+              <div class="row">
+                <div class="col-3">
+                  <p>
+                    SBI BITS achieves efficient resource deployment with a Canonical OpenStack implementation.
+                  </p>
+                  <p><a href="https://ubuntu.com/engage/sbi-casestudy">Ubuntu Pro&nbsp;›</a></p>
+                </div>
+              </div>
+            </div>
+          </div>
           {{ image (
-            url="https://assets.ubuntu.com/v1/7eebe11c-cisco-logo.png",
-            alt="Cisco",
-            width="288",
-            height="288",
+            url="https://assets.ubuntu.com/v1/80a56f7c-sbibits-photo-1558494949-ef010cbdcc31 1.png",
+            alt="",
+            width="1652",
+            height="612",
             hi_def=True,
-            loading="auto",
-            attrs={"class": "p-logo-section__logo"}
+            loading="auto|lazy"
             ) | safe
           }}
         </div>
-        <div class="p-logo-section__item">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/5588e80d-ibm-logo.png",
-            alt="IBM",
-            width="288",
-            height="288",
-            hi_def=True,
-            loading="auto",
-            attrs={"class": "p-logo-section__logo"}
-            ) | safe
-          }}
+      </div>
+    </div>
+  </div>
+</section>
+<section class="u-fixed-width p-section">
+  <hr class="p-rule is-dark is-bold">
+  <h2>
+    <strong>Accelerating innovation</strong>
+    <br>
+    We partner with the biggest names in tech to amplify the impact of open source.
+  </h2>
+</section>
+<section class="p-block">
+  <div class="u-fixed-width">
+    <hr class="p-rule is-dark">
+  </div>
+  <div class="row">
+    <div class="col-3 col-medium-2">
+      <h3 class="p-text--small-caps">Cloud</h3>
+    </div>
+    <div class="col-9 col-medium-4">
+      <h4 class="p-heading--2">Streamline your workflows and get the scalability you need with Ubuntu across clouds.</h4>
+      <div class="row">
+        <div class="col-2 col-medium-1 col-small-1">
+          
+  <img src="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_474,h_474/https://assets.ubuntu.com/v1/08d2774c-google-cloud-logo.png" srcset="https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_948,h_948/https://assets.ubuntu.com/v1/08d2774c-google-cloud-logo.png 2x" alt="" width="474" height="474" loading="auto|lazy">
```

Comment:
These are not proper assets, please don't copy/paste asset links directly from the browser. Search for them on the asset server or extract the asset url from the link. I believe the asset you need here is  https://assets.ubuntu.com/v1/08d2774c-google-cloud-logo.png. Also, please do not try to use both auto and lazy loading. It's auto when it's the first or only image, lazy everywhere else